### PR TITLE
Single to array coercion

### PR DESCRIPTION
**Pull request by cjllanwarne**

### CODE REVIEW

**Comment by mcovarr** [comment]
:+1:

**Comment by mcovarr** [comment]
The wheel chooses @Horneth

**Comment by Horneth** [comment]
Could be nice to also add a test in `WdlArrayTypeSpec` which already has some coercion unit tests.

**Comment by Horneth** [comment]
:+1:

File: src/test/scala/cromwell/SingleToArrayCoercionSpec.scala [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package cromwell
+
+import akka.testkit.EventFilter
+import cromwell.binding.values.{WdlString, WdlFile}
+import cromwell.util.SampleWdl
+
+class SingleToArrayCoercionSpec extends CromwellTestkitSpec("ContinueOnReturnCodeSpec") {
```
Comment (Horneth) [inline src/test/scala/cromwell/SingleToArrayCoercionSpec.scala]: `ContinueOnReturnCodeSpec` bad copy/paste :p

Comment (cjllanwarne) [inline src/test/scala/cromwell/SingleToArrayCoercionSpec.scala]: Added!

**Pull request merged**